bucket/kdiff3.json

```diff
@@ -1,23 +1,37 @@
 {
-    "version": "0.9.98",
-    "description": "A file and directory diff and merge tool",
-    "homepage": "http://kdiff3.sourceforge.net/",
+    "version": "1.8.5",
+    "description": "Utility for comparing and merging files and directories",
+    "homepage": "https://invent.kde.org/sdk/kdiff3",
     "license": "GPL-2.0-only",
     "architecture": {
         "64bit": {
-            "url": "https://downloads.sourceforge.net/project/kdiff3/kdiff3/0.9.98/KDiff3-64bit-Setup_0.9.98-2.exe#/dl.7z",
-            "hash": "sha1:4b7358aec8de6a3e719065fce37dd966c92ed3d4"
-        },
-        "32bit": {
-            "url": "https://downloads.sourceforge.net/project/kdiff3/kdiff3/0.9.98/KDiff3-32bit-Setup_0.9.98-3.exe#/dl.7z",
-            "hash": "sha1:9d7ec4fb0e2b83a321cb77ce47c86fd42fc8ec2b"
+            "url": "https://download.kde.org/stable/kdiff3/kdiff3-1.8.5-windows-64-cl.exe#/dl.7z",
+            "hash": "c0be76353161af622154993ed464a168a5cda38639deca76befe50f48e348ef5"
         }
     },
-    "bin": "kdiff3.exe",
+    "pre_install": [
+        "Expand-7zipArchive \"$dir\\kdiff3*.7z\" -Removal",
+        "Remove-Item \"$dir\\`$*\", \"$dir\\uninst*\", \"$dir\\7za.exe\" -Recurse"
+    ],
+    "bin": "bin\\kdiff3.exe",
     "shortcuts": [
         [
-            "kdiff3.exe",
+            "bin\\kdiff3.exe",
             "KDiff3"
         ]
-    ]
+    ],
+    "checkver": {
+        "url": "https://download.kde.org/stable/kdiff3/?C=M;O=D",
+        "regex": "kdiff3-(([\\d.]+))-windows-64"
```

Comment:
shouldn't this be "kdiff3-(([\\d.]+))-windows-64**-cl**"?

1.8.3 and also 1.8.4 had no "-cl", so who knows how the next version would show up.... so this would match in "checkver" but fail later in the "autoupdate" for the next version without a "-cl", right?!

Reply:
I was doing like that at first, but I changed because @Ash258 says: "Not a problem." https://github.com/lukesampson/scoop-extras/pull/5590#discussion_r571565144
Should I revert?

Reply:
oh, sorry, haven't seen this...

As @Ash258 is much more experienced, I'd say you better follow his advice...

Reply:
... and btw.,. **_THANK YOU_** for this manifest. I like Kdiff3 (and scoop ;) ) a lot and I'm very happy it' development continues _AND_ with scoop I'll always be up2date :)